projects/angular/src/data/tree-view/models/tree-node.model.spec.ts

```diff
@@ -152,5 +152,20 @@ export default function (): void {
       child.toggleSelection(true);
       [...root.children, ...child.children].forEach(n => expect(n.selected.value).toBe(ClrSelectedState.UNSELECTED));
     });
+
+    it('Re enabled disabled root node do not change children disable status', function () {
+      child.disabled = true;
+
+      expect(root.disabled).toBe(false);
```

Comment:
```suggestion
      expect(root.disabled).toBeFalse();
```

The code should be consistent.

Reply:
Corrected in commit 666bbec